Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -35,6 +35,78 @@
 
 static const wchar_t* TAG = L"CGPath";
 
+using namespace Microsoft::WRL;
+
+class _CGPathCustomSink : public RuntimeClass<RuntimeClassFlags<RuntimeClassType::WinRtClassicComMix>, ID2D1SimplifiedGeometrySink> {
+protected:
+    InspectableClass(L"Windows.Bridge.Direct2D._CGPathCustomSink", TrustLevel::BaseTrust);
+
+public:
+    ID2D1GeometrySink* getBackingSink() {
+        return geometrySink.Get();
+    }
+
+    _CGPathCustomSink(_In_ ID2D1GeometrySink* sink) : geometrySink(sink), isFigureOpen(false) {
+    }
+
+    STDMETHOD_(void, _EndFigure)(D2D1_FIGURE_END figureEnd) {
```

Comment:
are these necessary? In CoreText we just did void STDMETHODCALLTYPE fun()... type stuff